handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/filter/IsolationDiscoveryFilter.java

```diff
@@ -75,6 +82,9 @@ public IsolationDiscoveryFilter() {
       boolean newValue = emptyProtection.get();
       LOGGER.info("{} changed from {} to {}", EMPTY_INSTANCE_PROTECTION, emptyProtection, newValue);
     });
+    if (invocationFinishListenerRegistered.compareAndSet(false, true)) {
```

Comment:
Is this code not used now? I think it's better delete it.

Reply:
Oh, sorry. It has been fixed.